jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/hive/HiveSqlGenerationVisitorTest.java

```diff
@@ -0,0 +1,199 @@
+package com.exasol.adapter.dialects.hive;
+
+import com.exasol.adapter.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.metadata.*;
+import com.exasol.adapter.sql.*;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.*;
+import org.junit.jupiter.params.provider.*;
+import org.mockito.*;
+import utils.*;
+
+import java.sql.*;
+import java.util.*;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static utils.SqlNodesCreator.createSqlFunctionScalarWithTwoStringArguments;
+
+class HiveSqlGenerationVisitorTest {
+    private SqlNodeVisitor<String> visitor;
+    @Mock
+    private Connection connectionMock;
+
+    @BeforeEach
+    void beforeEach() {
```

Comment:
I'd like to discuss it. I don't really think it's a good idea, because it's only make code more complicated and `BaseTest` class is very confusing as it's not even a test class actually.

Reply:
Alright then, let's not spend the effort for now.